samples/stock/demo/black-and-white/demo.js

```diff
@@ -0,0 +1,218 @@
+            }, {
+                text: 'H1',
+                title: 'Set timeframe to 1 hour',
+                dataGrouping: {
+                    forced: true,
+                    units: [
+                        ['hour', [1]]
+                    ]
+                }
+            }, {
+                text: 'H4',
+                title: 'Set timeframe to 4 hours',
+                dataGrouping: {
+                    forced: true,
+                    units: [
+                        ['hour', [4]]
+                    ]
+                }
+            }, {
+                text: 'D1',
+                title: 'Set timeframe to 1 day',
+                dataGrouping: {
+                    forced: true,
+                    units: [
+                        ['day', [1]]
+                    ]
+                }
+            }, {
+                text: 'W1',
+                title: 'Set timeframe to 1 week',
+                dataGrouping: {
+                    forced: true,
+                    units: [
+                        ['week', [1]]
+                    ]
+                }
+            }, {
+                text: 'MN',
+                title: 'Set timeframe to 1 month',
+                dataGrouping: {
+                    forced: true,
+                    units: [
+                        ['month', [1]]
+                    ]
+                }
+            }],
+            selected: 4,
+            inputEnabled: false
+        },
+        series: [{
+            name: 'AAPL',
+            type: 'candlestick',
+            data: data,
+            color: '#000000',
+            lastPrice: {
+                enabled: true,
+                color: '#c0c0c0',
+                label: {
+                    enabled: true,
+                    backgroundColor: '#000000',
+                    padding: 2,
+                    shape: 'rect',
+                    borderRadius: 0
+                }
+            }
+        }],
+        tooltip: {
+            positioner: function () {
+                return {
+                    x: 10,
+                    y: 0
+                };
+            },
+            format: '{series.name} {point.open:.2f} {point.high:.2f} {point.low:.2f} {point.close:.2f}'
+        },
+        xAxis: {
+            crosshair: {
+                snap: false,
+                label: {
+                    enabled: true,
+                    format: '{value:%d %b %k:%M}'
+                }
+            },
+            startOnTick: true,
+            labels: {
+                distance: 4,
+                formatter() {
+                    const isAboveD1 =
+                        this.chart.series[0].currentDataGrouping.unitRange >=
+                        86400000;
+
+                    if (this.isFirst || isAboveD1) {
+                        return this.chart.time.dateFormat('%d %b %Y', this.value);
+                    }
+                    return this.chart.time.dateFormat('%d %b %k:%M', this.value);
```

Comment:
I believe the original sketch used time format `%e %b %k:%M`? The difference is the zero-padding on the date.